code/game/objects/items/pinpointer.dm

```diff
@@ -13,7 +13,7 @@
 	throw_speed = 3
 	throw_range = 7
 	materials = list(/datum/material/iron = 500, /datum/material/glass = 250)
-	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF
```

Comment:
Crew pinpointers shouldn't be indestructible, I made it so the nuke pinpointers are still indestructible. Without this change the unit test complains about crew pinpointers being in lockers that are lazilly loaded.